tox.ini

```diff
@@ -48,7 +48,6 @@ description = Enforce quality standards under {basepython} ({envpython})
 install_command = pip install {opts} {packages}
 commands =
   pre-commit run -a {posargs}
-  pylint {[base]pkg_name} tests --ignore=tm_tokenize
```

Comment:
If you remove this ignore, it must be moved under `.pylintrc`.

Reply:
Also, don't forget that it's configured under `lint-vetting` in a certain way, to keep that implemented.

Reply:
Doesn't the exclude in the pre commit yaml accomplish this?

Reply:
I know that some people integrated linters directly in their editors/IDEs. It would be frustrating if their experience is different. This is the same issue as with flake8. If it's invoked as a VS Code plugin, a contributor would have to know to reimplement settings. But if the config is in a standard location, it's sourced and the preferences are the same.

Reply:
Oh, and as for `exclude`, it mostly exists for pre-commit to decide whether to even invoke the hook when it's integrated as a pre-commit hook in the local Git repo.

Reply:
Oh.... Interesting, pylint ignoring these files is a side effect, but we should explicitly have pylint ignore them if it runs outside of precommit by a dev later?

Reply:
Adding ignores just here is ok. In fact my impression was that we want to address these in the future and avoid needing that exclude, right?

You can add them to pylintrc file if we do not plan to remove them, like https://stackoverflow.com/a/8427174/99834

Keep in mind that we need to pass config file to pylint when used as hook or we may be surprised it did not load the config. If you check my PRs you will see that I added it, and that ensures they both use same config.

@cidrblock I need to warn you that is a very small set of cases where adding an exclude in linter config own config is not enough when running linter under pre-commit and thus forcing us to have two sets of excludes. I think it happens very few times, but I just wanted to tell you. I remember that was needed for ansible-lint when the hook definition was with default `pass_filenames: true`. At some point I changed it to `pass_filenames: false` which basically makes pre-commit run it only once instead of once per file. That sorted the problem for ansible-lint and users had to add excludes in a single place. That was also a huge performance benefit as linter has a considerable long startup time due to ansible code... as in making it 9.5x faster.